src/components/Settings/ChangePassword.vue

```diff
@@ -59,14 +59,13 @@ export default {
     }
   },
   methods: {
-    save () {
+    async save () {
```

Comment:
This new implementation of the `save` method no longer triggers an event for the parent component to call the backend. The reason is that this doesn't work in combination with `async`/`await`. 
As a result, the `changePassword` method of the `Settings` component (the parent component) is no longer used.
Do you consider this bad style? Is there a nicer way of clearing the password fields?